pyworkflow/em/packages/gautomatch/convert.py

```diff
@@ -145,38 +144,57 @@ def writeSetOfCoordinates(workDir, coordSet, isGlobal=False):
 _rlnAnglePsi #3
 _rlnClassNumber #4
 _rlnAutopickFigureOfMerit #5
-"""
+    """
```

Comment:
minor fix: no need for tab here, otherwise the first line after header is shifted